setup.py

```diff
@@ -50,7 +50,7 @@ def read(*names, **kwargs):
 
 setup(
     name='fast-numpy-loops',
-    version='0.0.0',
+    version='0.0.6',
```

Comment:
Shouldn't this be 11 now?

Reply:
We tagged a bunch of fake releases while working on the rever github release. I've removed those now. I'll revert this back to `v0.1.0` which is the only real release at this point.